### PR TITLE
Task/005 profile image style

### PR DESCRIPTION
**Pull request by elanghel**

### CODE REVIEW

File: GithubPractice/ViewController.swift [hunk 1]
```diff
@@ -34,19 +34,19 @@ class ViewController: UIViewController {
             jobPositionField.text = job
         }
         
+        profileImage.layer.cornerRadius = profileImage.frame.height/2
+       // profileImage.clipsToBounds = true
```
Comment (vtoloconnicov) [inline GithubPractice/ViewController.swift]: remove the comment

File: GithubPractice.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -437,6 +437,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
```
Comment (vtoloconnicov) [inline GithubPractice.xcodeproj/project.pbxproj]: remove this file from commit

File: GithubPractice/ViewController.swift (hunk 1 above)
Comment (elanghel) [reply GithubPractice/ViewController.swift]: okay

File: GithubPractice.xcodeproj/project.pbxproj (hunk 2 above)
Comment (elanghel) [reply GithubPractice.xcodeproj/project.pbxproj]: sorry, one more commit

**Review by vtoloconnicov — APPROVED** [review]

**Pull request merged**